features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,13 @@
+When("I'm on the landing page") do
+  visit root_path
+end
+
+Then("I should see {string}") do |content|
+  expect(page).to have_content content
+end
```

Comment:
Move the assertion to a separate step definition file perhaps called assertions?